.github/workflows/r_valgrind.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     container: wch1/r-debug
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v1
+        uses: actions/checkout@v2.3.4
```

Comment:
/gha run r-valgrind

Reply:
looks like valgrind is happy with this change. I did check before that the image we use for valgrind tests had a recent version of `git` in it, and it does.

https://github.com/microsoft/LightGBM/runs/1349685476?check_suite_focus=true